Cargo.toml

```diff
@@ -48,6 +49,7 @@ wiremock = "0.5.3"
 [features]
 default = ["native-tls"]
 rustls = ["rustls-tls"]  # Leagcy support (<=0.17.0)
+trace = ["tracing"]
```

Comment:
This shouldn't be needed as by default making tracing optional creates a feature called `tracing`.
```suggestion
```

Reply:
I decided to just remove the feature and make it part of the library. Would that be OK?